app/js/models/addon_generator.js

```diff
@@ -45,6 +45,7 @@ var mo = new MutationObserver(function() {
   if (newEl !== el) {
     el = newEl;
     setTimeout(exec, 1);
+    mo.disconnect();
```

Comment:
Is this always valid? Are there scenarios where we may want to leave the MO? I can't think of any, but it was just something that I was considering when I suggested doing this.